chart/templates/olm/consoleclidownload.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: console.openshift.io/v1
```

Comment:
I don't know how this works, could you please explain?

Reply:
When the operator is installed, this customizes the Web Console adding links from where you can download the binaries (cf. [Customizing the Console doc](https://docs.openshift.com/container-platform/4.17/web_console/customizing-the-web-console.html#creating-custom-CLI-downloads_customizing-web-console)).
I attached a GIF to the PR description to see how you can get the new web console cli download page.

Reply:
Thanks